### ISSUE TITLE
[Bug] Export-DbaScript return 2 spaces if object doesn't exist

### ISSUE THREAD

**Issue by asnest** [sqlcollaborative/dbatools#5418]
### Before submitting a bug report:

- [X] *Running latest release of dbatools*
- [X] Verified errors are not related to permissions
- [ ] Is this bug with `Copy-DbaDatabase`? You can replicate it using `Backup-DbaDatabase ... | Restore-DbaDatabase ...`

> NOTE: `Copy-DbaDatabase` will not work in every environment and every situation. Instead, we try to ensure Backup & Restore work in your environment.

## Environmental data

```
### PowerShell version:

Name                           Value
----                           -----
PSVersion                      5.1.14393.2879
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.14393.2879
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1




### dbatools Module version:


Name    : dbatools
Path    : C:\Program Files\WindowsPowerShell\Modules\dbatools\0.9.811\dbatools.psd1
Version : 0.9.811

```

### SQL Server: 

```sql
Microsoft SQL Server 2016 (RTM-GDR) (KB4019088) - 13.0.1742.0 (X64)   Jul  5 2017 23:41:17   Copyright (c) Microsoft Corporation  Enterprise Edition: Core-based Licensing (64-bit) on Windows Server 2016 Datacenter 6.3 <X64> (Build 14393: ) 
```

## Errors Received

```powershell
no errors
```

## Steps to Reproduce

```sql
/*
    Any T-SQL commands involved or used to produce test objects/data.
*/
```

```powershell

$options = New-DbaScriptingOption
$options.IncludeIfNotExists = $true
$options.PrimaryObject = $false
$options.DriChecks = $true
$options.DriForeignKeys = $true
$options.DriWithNoCheck = $true
foreach ($Table in Get-DbaDbTable -ServerInstance $server -Database $database)
{if ($Table.name -NotLike 'MS*')
{$Table | Export-DbaScript -Path objects.sql -ScriptingOptionsObject $options -NoPrefix -Append}
}

```

## Expected Behavior
creating object script on each row in file

## Actual Behavior
if some tables don't have objects I want to script, function return 2 spaces. And file becomes look like zebra - code, some pages with spaces, code again

**Comment by potatoqualitee** [comment]
thanks for the report, we'll take a look when we have a moment

**Comment by sqllensman** [comment]
Issue was being caused by using a default value for the Prefix of "" even when -NoPrefix is specified.

Will submit fix along with some other updates to Export scripting functions

**Comment by potatoqualitee** [comment]
awesome! very much appreciated, @sqllensman

**Comment by potatoqualitee** [comment]
this will be resolved in dbatools 1.0, due in 20 days👍
